includes/embeds/class-amp-twitter-embed.php

```diff
@@ -299,13 +327,11 @@ private function sanitize_embed_script( $node ) {
 		}
 
 		// Handle case where script is immediately following.
-		$is_embed_script = (
-			$next_element_sibling
+		$is_embed_script = ( $next_element_sibling
 			&&
 			'script' === strtolower( $next_element_sibling->nodeName )
 			&&
-			false !== strpos( $next_element_sibling->getAttribute( 'src' ), $script_src )
-		);
+			false !== strpos( $next_element_sibling->getAttribute( 'src' ), $script_src ) );
```

Comment:
Is this change necessary?

Reply:
This was my auto-formatter in VS Code.

Reply:
A slight difference in the linter interpretation of the standards, both are valid it seems.